packages/plugins/color-picker/package.json

```diff
@@ -30,7 +31,18 @@
     "react-router-dom": "5.3.4",
     "styled-components": "5.3.3"
   },
+  "files": [
+    "./dist",
+    "./admin",
+    "strapi-admin.js",
+    "strapi-server.js"
+  ],
   "scripts": {
+    "build": "run -T tsc -p server/tsconfig.json --outDir ./dist/server",
+    "build:ts": "run build",
+    "watch": "run -T tsc -w --preserveWatchOutput",
```

Comment:
Would this be a good opportunity to discuss how we plan to handle this commands in the (near) future?

e.g. should we be doing `build:server` and `build:admin` etc?

Reply:
Sure why not. We probably will have to decide when we do pre-bundling and update everything else accordingly. 

I would probably go with some scripts instead of many npm commands. That will also make things easier to pull into that shared strapi-scripts package (whatever name it will have) later I think